src/main/webapp/app/course/manage/course-management-tab-bar/course-management-tab-bar.component.ts

```diff
@@ -198,4 +205,70 @@ export class CourseManagementTabBarComponent implements OnInit, OnDestroy, After
         const courseManagementRegex = /course-management\/[0-9]+(\/edit)?$/;
         return courseManagementRegex.test(this.router.url);
     }
+
+    private setExistingSummaryEntries() {
+        const numberRepositories =
+            this.course?.exercises
+                ?.filter((exercise) => exercise.type === 'programming')
+                .map((exercise) => exercise?.numberOfParticipations ?? 0)
+                .reduce((a, b) => a + b, 0) ?? 0;
```

Comment:
Please use descriptive variable names, they might not need to be as long as the ones that I proposed, but `a` and `b` are no good variable names 😅  

```suggestion
                .reduce((repositorySum, numberOfParticipationsForRepository) => repositorySum + numberOfParticipationsForRepository, 0) ?? 0;
```

Reply:
I see your point, for reduce to sum it might be debatable ^^